R/zzz.R

```diff
@@ -23,7 +22,16 @@ get_storage_dir <- function() {
   options()$staged.dependencies._storage_dir
 }
 
+set_storage_dir <- function(storage_dir) {
+  options("staged.dependencies._storage_dir" = storage_dir)
+}
+
 .onLoad <- function(libname, pkgname) {
+
+  storage_dir <- getOption(
+    "staged.dependencies._storage_dir",
+    path.expand("~/.staged_dependencies")
```

Comment:
Should this be .staged.dependencies though - so we don't change this?

Reply:
Ah yes! I think I'm just too used to underscores :) Will fix.